### DISCUSSION TITLE
can not use  the button（search）

### DISCUSSION THREAD

**Discussion by JessBobby** [community/community#63138] [Mobile] [answered]
No reaction when clicking the button of search.
device ：Android 13 （MIUI14）
version：1.124.0

**Comment by alcere** [comment]
Hi @JessBobby! We are sorry to heard that! Could you share more details, please? Where is the global search button you see in Home the one failing for you? Thanks!

**Reply by JessBobby** [reply, marked as answer]
Hi，yesterday，I had the problem at the homepage .The global search failed，and when i found  the page of feedback,it failed too.
**The problem has been solved by updating the newest version 1.125.0.
thanks！**
as someone says，I am in China.Maybe the Internet cause overtime request.
the second one I think is my device problem .My device cannot install apks file and use Google service.the same problem would occur  when I update by using a software from the apks file in a mirror Google store

**Comment by CodeMaster7000** [comment]
Hi @JessBobby 

If the GitHub search function is frozen, it may be due to the server request timing out. As GitHub keeps its search function quick for users, it limits the amount of time each query can run before timing out - helping conserve bandwidth on the site. To try to fix it, check your internet connection and retry later.

Alternatively, it could be that your phone is running slow - so backing up files to create storage space and restarting it may help.

**Reply by JessBobby** [reply]
The problem has been solved by updating the newest version 1.125.0.
thanks！
I will use the suggestion you mentioned to solve when I meet similar problems.
